MetaMask/metamask-extension
# feat: add brand evo font files

## **Description**

This PR adds the new brand evolution font files to the portfolio and ensures they are available in the styles used across the MetaMask extension. The specific font files added are:

- `MM_Sans_Mono-Variable.woff2` (Default weight 400) Supports 400, 500, 600, 700, 800, 900
- `CentraNo1-Book.woff2` (weight 400)
- `CentraNo1-Medium.woff2` (weight 500)
-  `CentraNo1-Bold.woff2` (weight 700)

These fonts have been integrated into the relevant CSS configuration to ensure they are correctly applied within the extension's design components, aligning with the brand's updated visual guidelines.

The reason for this change is to incorporate the latest assets into the extension, providing a consistent visual experience across all touchpoints. The solution involved adding the font files to the appropriate directory in the codebase and updating the CSS to reference these new fonts with the specified class names.

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/26672?quickstart=1)

## **Related issues**

Fixes: #756

## **Manual testing steps**

1. Open the extension and navigate to pages where these fonts are applied.
2. Inspect the elements to confirm that the `isBrandEvolution` class is applied and the fonts are rendering correctly.
3. Perform cross-browser testing to ensure the fonts display correctly on all supported browsers.
4. Conduct tests on different devices to verify that the fonts are rendered as expected.

## **Screenshots/Recordings**

### **Before**



### **After**
<img width="755" alt="Screenshot 2024-08-29 at 4 31 36 PM" src="https://github.com/user-attachments/assets/f663b5f4-5af0-4de2-b5d3-de2e63f69383">

https://github.com/user-attachments/assets/62614eee-d7bf-4bbd-bd20-3bf59024da06

https://github.com/user-attachments/assets/be1bc793-161f-45c5-956a-6eff27ff61f5

https://github.com/user-attachments/assets/c7298b5f-d343-4d67-bf6a-0dc4577707bf

https://github.com/user-attachments/assets/04961768-d1f2-4e3e-ac99-53131dce3071





## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
